debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -1568,9 +1569,10 @@ public void abandonTransactions(Duration retention) throws InterruptedException
                                     }
                                     first = false;
                                 }
-                                LOGGER.warn("Transaction {} (start SCN {}, change time {}, redo thread {}, {} events) is being abandoned.",
+                                LOGGER.warn("Transaction {} (start SCN {}, change time {}, redo thread {}, {} events{}) is being abandoned.",
```

Comment:
There is missing space before `{}` placeholder. Is it intentional?

Reply:
It is, when the entry has a value it begins with `, ` to be consistent with other listed entries.